components/fs/fat/FATFS_README.md

```diff
@@ -9,51 +9,48 @@ ASYNCFATFS provides and relies on a fully asynchronous file system API and block
 To initiate a request to the file system, the client must enqueue one or more requests using the interfaces defined in fs_protocol.h, and notify the file system accordingly. The asynchronous nature ensures that the client does not need to block while waiting for the operation to complete.
 
 # How FATFS Is Modified to Be Asynchronous
-The asynchronous design of ASYNCFATFS is very similar to Linux’s io_uring, which efficiently handles non-blocking I/O operations using an event-driven model. The system follows a coroutine-based event-worker design, where requests are processed without blocking, and I/O operations are handled asynchronously.
+The asynchronous design of ASYNCFATFS is very similar to Linux’s io_uring, which efficiently handles non-blocking I/O operations using an event-driven model. The system follows a thread-based event-worker design, where requests are processed without blocking, and I/O operations are handled asynchronously.
```

Comment:
Why change this to thread-based? Wouldn't coroutine-based be more accurate?

Reply:
I just looked at the commit message which answers my question. Although we need to be clear somewhere that these are not kernel-threads. Perhaps at the top of the README it can say that when we talk about 'threads' we are talking about co-operative multithreading.